test/rest_api/run

```diff
@@ -2,9 +2,7 @@
 
 import sys
 
-# Use the run.py library from ../cqlpy:
-sys.path.insert(1, sys.path[0] + '/test/cqlpy')
-import run
+from test.cqlpy import run
```

Comment:
I'm confused how this path works if I do something like
```
cd scylla
./test/rest_api/run
```
Does it know that "test.cqlpy" begins two directory levels up? How?